script.js

```diff
@@ -1 +1,152 @@
+//yehey 하이
+const todoForm = document.querySelector(".submitContainer"),
+    todoInput = todoForm.querySelector("input"),
+    todoList = document.querySelector(".todoListContainer"),
+    completeList = document.querySelector(".completeContainer");
 
+const TODOS_LS = "todos";
+let todos =[];    //여러개의 할 일들을 담을 배열
+const COM_LS = 'done';
+let done=[];
+
+function loadToDos(){
+    const localTodo = localStorage.getItem(TODOS_LS);
+    const localDone = localStorage.getItem(COM_LS);
+    if(localTodo!==null){
+        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
+        const local = JSON.parse(localTodo);
+        local.forEach(function(toDo){ addToDo(toDo.text);})
```

Comment:
Javascript es6문법을 활용해주시면 좋을 것 같습니다~!
```suggestion
        local.forEach((toDo) => {
            addToDo(toDo.text)
        })
```